x-pack/plugins/security_solution/public/entity_analytics/components/risk_summary_flyout/risk_summary.tsx

```diff
@@ -81,7 +81,7 @@ const FlyoutRiskSummaryComponent = <T extends RiskScoreEntity>({
 
   const xsFontSize = useEuiFontSize('xxs').fontSize;
 
-  const columns = useMemo(() => buildColumns(), []);
+  const columns = useMemo(() => columnsArray, []);
```

Comment:
no need for the `useMemo` now, we can just use the columns directly

Reply:
Will get that patched, ty